utils/authorization.js

```diff
@@ -0,0 +1,11 @@
+import { Meteor } from 'meteor/meteor';
+
+const authCheck = (collection, userId, groceryId) => {
+	// Make sure only the owner can delete
+	const grocery = collection.findOne(groceryId);
+	if (grocery.owner !== userId) {
+		throw new Meteor.Error('not-authorized');
+	}
+}
```

Comment:
These references to grocery should probably be changed to something more generic